src/routes/+page.svelte

```diff
@@ -1,16 +1,5 @@
 <script>
     export let data
-    import { AppBar, AppShell } from '@skeletonlabs/skeleton'
 </script>
 
-<AppBar
```

Comment:
Since we want the AppBar to show up on every page, I've moved it to the `+layout.svelte` file, which is applied to the current and all child directories